stock_picking_report_valued/models/stock_picking.py

```diff
@@ -41,12 +41,30 @@ def _compute_amount_all(self):
         records...).
         """
         for pick in self:
-            amount_untaxed = sum(pick.move_line_ids.mapped(
-                'sale_price_subtotal'))
-            amount_tax = sum(pick.move_line_ids.mapped(
-                'sale_price_tax'))
+            round_curr = self.sale_id.currency_id.round
+            amount_untaxed = amount_tax = 0.0
+            for tax_id, tax_group in self.get_taxes_values().items():
```

Comment:
Maybe this computation on rounding should depends on config parameter (round per line or round globally) as the result should be different.

Reply:
Changes done